### DISCUSSION TITLE
24.10 Orangepi5plus no App Center

### DISCUSSION THREAD

**Discussion by radekolszak** [Joshua-Riek/ubuntu-rockchip#1118] [Q&A]
Hello. I've installed 24.10 desktop on orangepi 5 plus but there is no app center on desktop dock and in apps . in am typing app center in search and nothing. How to install app center.? Also there is no web browser installed.
Edit .I have installed App Center using command `sudo snap install --edge snap-store`

**Comment by Joshua-Riek** [comment]
I can't include many default applications due to GitHub's 2GB artifact limit. You did not read https://github.com/Joshua-Riek/ubuntu-rockchip/discussions/1104.

It makes me honestly sad I cannot distribute the software I want to give users a true and full Ubuntu experience. If you would like these sorts of applications, then suggest a method to distribute large files to users for free. I'm a random person on the internet, I do not have the time, resources, or money to invest in general large file storage services for users to download artifacts.

**Comment by radekolszak** [comment]
Thank you for your work. Maybe host files using free google drive service. ?

**Reply by Joshua-Riek** [reply]
Lol, no.. there are storage limits, and that is sketchy as hell.
